recipe/gen_patch_json.py

```diff
@@ -630,6 +630,24 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
             if pversion < v2022_6_1:
                 record['depends'].append('tornado <6.2')
 
+        if record_name in {"distributed", "dask"}:
+            version = pkg_resources.parse_version(record["version"])
+            if (
+                version >= pkg_resources.parse_version("2021.12.0") and
+                version < pkg_resources.parse_version("2022.8.0") or
+                version == pkg_resources.parse_version("2022.8.0") and
+                record["build_number"] < 2
```

Comment:
Why these constraints?

Reply:
The oldest available `dask-core` nightlies that we would want to exclude are `2021.12.0a211220`:

https://anaconda.org/dask/dask-core/files?version=2021.12.0a211220

This targets all Dask packages from `2021.12.0` to the older builds of `2022.8.0`. Is there an easier way to target these packages, or do you think we should target more Dask releases?